assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -1168,25 +1263,24 @@ end subroutine nc_put_int_1d
 
 !--------------------------------------------------------------------
 
-subroutine nc_put_single_real_1d(ncid, varname, varindex, varval, context, filename)
+subroutine nc_put_real_0d(ncid, varname, varval, context, filename)
```

Comment:
In GitHub, this lines up with the wrong routine. For 0d variables, there is no 'start,count' so there is no need for a 'varindex'.